docs/static/breaking-changes.asciidoc

```diff
@@ -29,6 +29,9 @@ and have the persistent queue enabled, we strongly recommend that you drain or
 delete the persistent queue before you upgrade. See <<upgrading-logstash-pqs>>
 for information and instructions.
 
+=== Breaking changes in filebeat input plugin
+Please note that logstash with filebeat input plugin prior to version N.N.NNN is not compatible with filebeat version 6.3.0 or above. See [filbeat breaking change](https://www.elastic.co/guide/en/beats/libbeat/master/breaking-changes-6.0.html#breaking-changes-ls-index) for details.
+
```

Comment:
Hi @dilchenko 
- Here's the link format you need:
`{beats-ref}/breaking-changes-6.0.html#breaking-changes-ls-index[filebeat
breaking change]`
- Need a `[float]` tag above heading. (See other headings for example.)
- Need a return between heading and first line of text.
- Depending on what `version N.N.NNN` turns out to be, we may need to rearrange content a bit.  Possibly moving your note under 6.0.  We'll see. 

Please tag me after you update the version info, and I'll look again. 
In the meantime, let me know if you have questions or comments.  Thanks!